recipes/rp2biosensor/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "rp2biosensor" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brsynth/rp2biosensor/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c82362af98cfe3fecb67f236be2b7b7c482069f64db4d8e92bd87736052202a5
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install -vv .
+  entry_points:
+    - rp2biosensor = rp2biosensor.__main__:main
+
+requirements:
+  host:
+    - python >=3.7,<3.9
+    - pip
+  run:
+    - python >=3.7,<3.9
```

Comment:
Are these upper bounds necessary?

Reply:
Hi @jaimergp ,

Indeed, it's not necessary, we changed that to `python >=3.6`. Thank you.

Reply:
`setup.py` [says](https://github.com/brsynth/rp2biosensor/commit/44fc21c1f315a35a30067108f28751ff9d36655b#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R45) you need Python 3.7+. Which one is correct?

Reply:
@jaimergp , Python3.6+ is the correct one. We just modified the setup too. Thanks again.